src/manage.c

```diff
@@ -2461,6 +2463,15 @@ launch_osp_openvas_task (task_t task, target_t target, const char *scan_id,
     }
   cleanup_iterator (&families);
 
+  if (empty) {
+    if (error)
+      *error = g_strdup ("Exiting because VT list is empty");
```

Comment:
Maybe add a hint why this is happening (e.g. DB rebuild in progress, ...) so that one seeing this message actually knows what's going on?

Reply:
I'm not sure what to add. Since #1926 this should not happen during rebuild. It could happen on a fresh install, before the rebuild/update has happened, then the user must run greenbone-feed-sync, but the situation is different on GOS (I guess the devices ship with an initial feed and the feed sync is automatic).  Any suggestions?

Reply:
Sounds good.